includes/WebAuthn/class-cbor-decoder.php

```diff
@@ -0,0 +1,300 @@
+<?php
+
+if ( ! defined('ABSPATH') ) {
+	die('Bye!');
+}
```

Comment:
A minor nitpick: this file contains only class definitions; therefore, even if this file is accessed directly, there will be no errors/warnings. That's why it is safe to remove these lines.